ql/src/test/queries/clientpositive/complex_type_to_string.q

```diff
@@ -0,0 +1,31 @@
+create table table1(id int, txt string, num int, flag string);
+create table table2(id int, txt string, num int, flag boolean);
+
+insert into  table1 values
+  (1, 'one',   5, 'FALSE'),
+  (2, 'two',  14, 'TRUE'),
+  (3,  NULL,   3, 'FALSE');
+insert into  table2 values
+  (1, 'one',   5, 'FALSE'),
+  (2, 'two',  14, 'TRUE'),
+  (3,  NULL,   3, 'FALSE');
+  
+select cast(array(*) as string) from table1;
+select cast(array(*) as string) from table2;
```

Comment:
Could you please add these tests to [udf_to_string.q](https://github.com/apache/hive/blob/master/ql/src/test/queries/clientpositive/udf_to_string.q) file? I think we don't need a separate q file. WDYT?

Reply:
Added tests for complex types in udf_to_string.q